### DISCUSSION TITLE
Trying to change background color based on custom/script

### DISCUSSION THREAD

**Discussion by windowsrefund** [polybar/polybar#2974] [Q&A] [answered]
I'm running 3.6.3 and am having a hell of a time getting this little script to work. Essentially, I'm just checking if a file exists. If it does, change background color for %output% to red. Also, I can't understand why I appear to need to set `tail = true`. If I set it to false, I get a "blinking" behavior only when the script returns  a non-zero return code (despite the fact it echos as the documentation suggests).

The module
```
[module/foo]
type = custom/script
interval = 5
exec = ~/.config/polybar/scripts/foo.sh
tail = true
format = <label>
format-fail = <label-fail>
format-fail-background = #cc6666
format-prefix = foo
label = %output%
label-fail = %output%
; the following attempt also fails as a replacement for format-fail-background
; label-fail = %{B#cc6666}output%
```
The script
```
#!/usr/bin/env bash

if [ -f /tmp/foo ]; then
    echo 󱈸 
    exit 1
else
    echo  
fi
```

The more I mess around with this, I'm convinced the strange behavior relates in some way the quirky behavior of tail (mentioned above). If I set `tall = false`, I get my background color but then the entire module starts blinking (at each run interval) and I completely lose the prefix. It's as if the module is wiping the current output each time before it runs the script? Oddly, it doesn't behave in this quirky fashion when the script returns 0 so it's probably a bug?

Thanks in advance for any help.

**Comment by patrick96** [comment, marked as answer]
I think you are experiencing the same issue as #2636. This behavior has been changed for the next release.

**Comment by windowsrefund** [comment]
Yep, that looks like the issue. Looks like it was fixed back in April of 2022 though. I understand 3.7.0 has 4 open issues but are they important enough to hold back on the 33 that have been merged?

Thanks for the info.

**Reply by patrick96** [reply]
Right now I don't really have time to do any substantial work on polybar at all (for at least a few weeks), so it doesn't really matter if issues are holding up a release, I don't have time to publish a release anyway (especially considering that we will likely receive some bug reports within a week of the release).

The two tray issues really should get into the 3.7 release. The brightness and nerd font ones could reasonably be delayed and merged into a bugfix release.
